examples/simulink/qp_data.c

```diff
@@ -0,0 +1,682 @@
+/***************
```

Comment:
What about using the 'getting_started' QP example? This mass-spring data file is quite big